packages/maskbook/src/plugins/RedPacket/hooks/useAvailabilityComputed.ts

```diff
@@ -26,10 +26,10 @@ export function useAvailabilityComputed(account: string, payload: RedPacketJSONP
 
     const isEmpty = availability.balance === '0'
     const isExpired = availability.expired
-    const isClaimed = availability.ifclaimed
+    const isClaimed = availability.claimed_amount !== '0'
```

Comment:
Why not use `claimed`?

Reply:
`claimed_amount` works. `claimed` is newly added, I'm not sure if it works.